classes/Group/Group.mjs

```diff
@@ -233,8 +233,8 @@ export default class Group {
     }
 
     async update() {
-        this.validateGroup()
-        return database.update(this.data, process.env.TPENGROUPS)
+        await this.validateGroup()
+        return database.update({...this.data, type:"Group"}, )
```

Comment:
I assume we can eventually factor out the need for `type` here.  I believe we tried to factor that out.